[2024-10-16T09:55:49.643+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [queued]>
[2024-10-16T09:55:49.662+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [queued]>
[2024-10-16T09:55:49.663+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:49.663+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:55:49.665+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:49.689+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-03 00:00:00+00:00
[2024-10-16T09:55:49.697+0000] {standard_task_runner.py:55} INFO - Started process 8870 to run task
[2024-10-16T09:55:49.704+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-03T00:00:00+00:00', '--job-id', '1799', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp1_btn86u']
[2024-10-16T09:55:49.705+0000] {standard_task_runner.py:83} INFO - Job 1799: Subtask insert_rate
[2024-10-16T09:55:49.818+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:55:49.883+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:55:49.999+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-03T00:00:00+00:00
[2024-10-16T09:55:50.004+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.10300606, '2024-06-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:55:50.021+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:50.451+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:50.467+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.10300606, '2024-06-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:55:50.473+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:55:50.493+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240603T000000, start_date=20241016T095549, end_date=20241016T095550
[2024-10-16T09:55:50.625+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:55:50.662+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:05:48.990+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [queued]>
[2024-10-16T10:05:49.003+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [queued]>
[2024-10-16T10:05:49.004+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:49.004+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:05:49.005+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:49.022+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-03 00:00:00+00:00
[2024-10-16T10:05:49.027+0000] {standard_task_runner.py:55} INFO - Started process 11398 to run task
[2024-10-16T10:05:49.032+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-03T00:00:00+00:00', '--job-id', '2512', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpauhc9jdz']
[2024-10-16T10:05:49.033+0000] {standard_task_runner.py:83} INFO - Job 2512: Subtask insert_rate
[2024-10-16T10:05:49.133+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-03T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:05:49.185+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:05:49.264+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-03T00:00:00+00:00
[2024-10-16T10:05:49.266+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.10300606, '2024-06-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:05:49.280+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:49.587+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:49.599+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 447.10300606, '2024-06-03')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:05:49.604+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:05:49.624+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240603T000000, start_date=20241016T100548, end_date=20241016T100549
[2024-10-16T10:05:49.659+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:05:49.690+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
